CHANGELOG.md

```diff
@@ -18,6 +18,18 @@ For reference, the possible headings are:
 * **Known Issues**
 * **Notes**
 
+## [2.0 Beta 5] - 2018-08-01
+
+Tag name: v2.0.0b5
+
+### Changed
+
+* Supported version of Tendermint `0.22.3` -> `0.22.8`. [Pull request #2429](https://github.com/bigchaindb/bigchaindb/pull/2429).
+
+### Fixed
+
+* Raise DoubleSpend exception if transaction is already part of the database. [Pull request #2422](https://github.com/bigchaindb/bigchaindb/pull/2422).
```

Comment:
This isn't quite right. The DoubleSpend is now raised if there's any other transaction that spends the same output, _even if it has the same transaction ID_ as that "other" transaction.

Reply:
@kansi Is this right? Is this how you would describe the change?